src/popup/fxa.js

```diff
@@ -42,17 +67,16 @@ function setupAccountMenu(user) {
     if (emailElement) {
       emailElement.innerText = user.email;
 
-      if (user.avatar) {
-        document.getElementById("avatar").style.backgroundImage = `url("${user.avatar}")`;
-      } else {
+      if (user.avatarDefault) {
         document.getElementById("avatar").style.backgroundImage = `url("/icons/avatar.svg")`;
+      } else {
+        document.getElementById("avatar").style.backgroundImage = `url("${user.avatar}")`;
```

Comment:
When I've tested locally, I've encountered conditions where both `avatar` and `avatarDefault` are `undefined`. Wouldn't that lead to us setting an empty `backgroundImage` here?

Reply:
Strange, I haven't encountered that yet. I'll add a guard for it.